SonarRunner.Shim/Resources.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="MSG_CallingSonarRunner" xml:space="preserve">
-    <value>Calling the sonar-runner...</value>
+    <value>Calling the sonar-scanner...</value>
```

Comment:
as discussed, :+1: - note that we can still continue within the code to refer to it as the `sonar-runner` (i.e. we don't need to change class names and so on)